sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/CloudEvent.java

```diff
@@ -0,0 +1,325 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.messaging.eventgrid;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.serializer.json.jackson.JacksonJsonSerializerBuilder;
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.serializer.*;
```

Comment:
Generally I think we try to avoid * imports, I know I had to go dive into IntelliJ settings to make it stop doing this by default:

Settings > Editor > Code Style > Java > set both **count to use import with '*'** settings to **99**.